packit_service/worker/handlers/abstract.py

```diff
@@ -81,11 +90,15 @@ def _add_to_mapping(kls: Type["JobHandler"]):
 
 class Handler:
     triggers: List[TheJobTriggerType]
+    api: Optional[PackitAPI] = None
+    local_project: Optional[LocalProject] = None
+    _config: Optional[ServiceConfig] = None
 
-    def __init__(self, config: ServiceConfig):
-        self.config: ServiceConfig = config
-        self.api: Optional[PackitAPI] = None
-        self.local_project: Optional[LocalProject] = None
+    @property
+    def config(self) -> ServiceConfig:
```

Comment:
I would appreciate if we can be specific here. I am always wondering about type when I see only `config`.

```suggestion
    def service_config(self) -> ServiceConfig:
```

Reply:
I just wanted to be consistent as in `SteveJobs` class there it is also named as `config`, so should I change it everywhere?

Reply:
Other PRs authors' will not like us during rebase..;)

What about changing this one here and the one in `SteveJobs` in a separate PR so other's can rebase more easily?